opennlp-tools/src/main/java/opennlp/tools/postag/POSTagFormat.java

```diff
@@ -23,5 +23,5 @@
  */
 public enum POSTagFormat {
 
-  UD, PENN, UNKNOWN
+  UD, PENN, CUSTOM, UNKNOWN
```

Comment:
Just wanted to ask about the `CUSTOM` value and the `NoOp` class names.

Reply:
Would you prefer a no-op value here? I was thinking as a user wanting to have custom tags :)

Reply:
IMHO `NO_OP` would be adequate if the enum defines the "mapping" process.

But if it defines the "format" (I'd probably rather call it "tagset" than "format" then, but ok), then `CUSTOM` may be more sensible.

In the end, as long as it means the mapper can be turned off... it is probable fine.